src/pages/HowToUse/index.js

```diff
@@ -1,10 +1,31 @@
-import React from 'react'
+import React from 'react';
 import Box from '@material-ui/core/Box';
 import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
 import GenericHeaderSection from '../../components/GenericHeaderSection';
-import TagGeneratorInstructions from "../../components/TagGeneratorInstructions";
+import ImageComponent from '../../components/ImageComponent';
+import SettingsGearIcon from '../../icons/SettingsGearIcon';
+
+const useStyles = makeStyles((theme) => ({
+  gridContainer: {
+    padding: '64px 32px',
+    margin: 'auto',
+  },
+  typoStyle: {
+    fontWeight: '400',
+    [theme.breakpoints.down('xs')]: {
+      fontSize: '14px',
+    },
+  },
+  gridPadding:{
+    padding: '24px 16px 0'
```

Comment:
It's a minor issue, but eslint is showing trailing comma error here

Reply:
@bruceplai, fixed es lint warnings.